app/models/searchs/vacancy.rb

```diff
@@ -16,32 +14,18 @@ def list(query = nil, lang = nil)
         description,
         company_name,
         created_at,
-        slug,
-        document
-      FROM
-        (SELECT
-          id,
-          job_title,
-          location,
-          description,
-          company_name,
-          created_at,
-          slug,
-          to_tsvector('#{lang}', location) || ' ' ||
-          to_tsvector('#{lang}', company_name) || ' ' ||
-          to_tsvector('#{lang}', job_title) || ' ' ||
-          to_tsvector('#{lang}', description) AS document
-        FROM vacancies) vacancies
+        slug
+      FROM vacancies
       WHERE
-        vacancies.document @@ to_tsquery('#{lang}', ?)
+        vacancies.tsv @@ to_tsquery('portuguese', ?)
       ORDER BY created_at DESC
     }
```

Comment:
You can change this to use something like this `@repository.recent.where('tsv @@ to_tsquery('portuguese', ?)', 'bla')`. I think that this is better